k3s-leader/k3s-leader.md

```diff
@@ -0,0 +1,102 @@
+﻿![K3S logo](https://k3s.io/images/logo-k3s.svg "K3S Logo")
```

Comment:
You should talk a little about what a leader is and why you should have more than one